fulp_modules/features/antagonists/infiltrators/infil_objectives.dm

```diff
@@ -0,0 +1,204 @@
+/datum/antagonist/traitor/infiltrator/forge_traitor_objectives()
+	if(!employer)
+		return
+	switch(employer)
+		if("Corporate Climber")
+			forge_traitor_objectives_cc()
+		if("Animal Rights Consortium")
+			forge_traitor_objectives_arc()
+		if("Gorlex Marauders")
+			forge_traitor_objectives_gm()
+
+// Corporate Climber Objectives
+
+/datum/antagonist/traitor/infiltrator/proc/forge_traitor_objectives_cc()
```

Comment:
can this all be one proc with a switch()? seems weird to have forge traitor objectives all be separate.